packages/dds/tree/api-report/tree.api.md

```diff
@@ -146,8 +143,8 @@ export const Any: "Any";
 // @internal
 export type Any = typeof Any;
 
-// @beta
-type ApplyKind<T, Kind extends FieldKind_2> = Kind extends FieldKind_2.Required ? T : undefined | T;
+// @public
+export type ApplyKind<T, Kind extends FieldKind> = Kind extends FieldKind.Required ? T : undefined | T;
```

Comment:
There was an attempt to make "transitively public" types like 'ApplyKind' less discoverable by re-exporting them as a namespace (see  InternalClassTreeTypes below).

This ran into a couple of issues.  One is that the produced rollups "resolve" these to the non-namespaced / non-exported declarations.  The second is that there is no place to put a release tag (e.g., mark as public), which triggers lint errors when API-Extractor merges rollups for the 'fluid-framework' package:
```
fluid-framework: Error: /Users/danlehen/gh/f0/packages/dds/tree/dist/tree-public.d.ts:527:1 - (ae-missing-release-tag) "InternalClassTreeTypes" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```